### PR TITLE
avoid using six.string_types in settingshelper

### PR DESCRIPTION
**Pull request by nickpell**

### CODE REVIEW

**Review by gbova — COMMENTED** [review]
What does this do exactly?

File: settingshelper.py [hunk 1]
```diff
@@ -139,10 +139,10 @@ def make_couchdb_tuples(self):
         return [self._make_couchdb_tuple(row) for row in self.couchdb_apps]
 
     def _make_couchdb_tuple(self, row):
-        if isinstance(row, six.string_types):
-            app_label, postfix = row, None
-        else:
+        if isinstance(row, tuple):
```
Comment (millerdev) [inline settingshelper.py]: Could make this a little more resilient to custom values in localsettings (`LOCAL_COUCHDB_APPS`) with `isinstance(row, (tuple, list))`

**Review by millerdev — APPROVED** [review]

File: settingshelper.py (hunk 1 above)
Comment (nickpell) [reply settingshelper.py]: Actually it looks like ```LOCAL_COUCHDB_APPS``` is never used, and never including in localsettings in ```commcare-cloud```.  I'm going to remove it.

Comment (nickpell) [reply settingshelper.py]: (Although I see your point about being resilient to ```list```s)

**Comment by nickpell** [comment]
@gbova This function iterates through values of ```COUCHDB_APPS```.  Sometimes, it is a tuple, other times it is a unicode value.  When it's a tuple the values have to be unpacked.  Since ```six.string_types``` is ```(unicode, bytes)``` in python 2 and ```(unicode,)``` in python 3, I want to be sure that the change in behavior won't mess anything up.  I'm trying to removing ```six.string_types``` when possible and when I'm not sure, include soft asserts to check that the value is really ```unicode```.

**Comment by gbova** [comment]
@nickpell Thanks for the explanation!

**Review by gbova — APPROVED** [review]

**Pull request merged**